### PR TITLE
Spelling and misc fixes

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Name=eth0
 [Network]
 Address=$IP_OF_MENDER_CLIENT
 Gateway=$IP_OF_MENDER_SERVER_FROM_DEVICE
-" | mender-artifact cp <imgname>.sdimg:/etc/systemd/network/eth/network
```
Comment (oleorhagen) [inline 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: good catch! Can't trust me eyes these days, appareantly

File: 04.Artifacts/99.Variables/docs.md [hunk 2]
```diff
@@ -336,12 +336,12 @@ Set this variable in `local.conf` in order to make the device recognize the orga
...
 #### MENDER_UBI_LEB_PEB_BLOCK_OVERHEAD
 
-The overhead that each logical erase block (LEB) of the UBI device adds to the physical erase block (PEB), in bytes. In other words, how many bytes are "wasted" in each LEB. Usually set automatically, but can be overriden.
+The overhead that each logical erase block (LEB) of the UBI device adds to the physical erase block (PEB), in bytes. In other words, how many bytes are "wasted" in each LEB. Usually set automatically, but can be overridden.
 
 
 #### MENDER_UBI_LEB_SIZE
 
-The size of each logical erase block (LEB) on the UBI device, in bytes. Usually set automatically from the various `MENDER_UBI_*_OVERHEAD` variables, but can be overriden.
+The size of each logical erase block (LEB) on the UBI device, in bytes. Usually set automatically from the various `MENDER_UBI_*_OVERHEAD` variables, but can be overridden.
```
Comment (oleorhagen) [inline 04.Artifacts/99.Variables/docs.md]: Really looks like we should start running ispell on these things, before merging!

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ MENDER_IMGPATH=<sdimg>
 ```
 And then running:
 ```bash
-mender-artifact cat $MENDER_IMGPATH:/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf | sed 's#[@]MENDER_DEMO_WIFI_PASSKEY[@]#$NW_PASSWORD#' | sed 's#[@]MENDER_DEMO_WIFI_PASSKEY[@]#NW_SSID#' | mender-artifact cp $MENDER_IMGPATH:/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf"
```
Comment (oleorhagen) [inline 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: If Changing this, the keyword `MENDER_DEMO_WIFI_PASSKEY` has to change in the corresponding conf file as well.

File: .ispell_english [hunk 4]
```diff
@@ -669,14 +688,16 @@ IDs
 ieee
 ies
 ietf
+ifdefs
 ifxbR
 iJy
 ILg
 IMAGECMD
-IMAGETYPE
```
Comment (oleorhagen) [inline .ispell_english]: What is your rule of thumb for caps vs nocaps here?

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md (hunk 3 above)
Comment (oleorhagen) [reply 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: that is true! thank you. Slipped past me twice!

**Comment by drewmoseley** [comment]
@oleorhagen so are you ok with me merging this now?

**Pull request merged**